server/seeders/20170317175000-DocumentsSeeder.js

```diff
@@ -0,0 +1,47 @@
+const faker = require('faker');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('documents', [
```

Comment:
This can be DRYed up by mapping over an array of `ids` and producing objects, since the id is the only thing that changes in these documents.